tests/cpptests/test_cpp_llapi.cpp

```diff
@@ -883,6 +883,11 @@ TEST_F(LLApiTest, testGetters) {
   ASSERT_EQ(0.42, RediSearch_IndexGetScore(index));
   ASSERT_STREQ("yiddish", RediSearch_IndexGetLanguage(index));
 
+  RSDoc* d = RediSearch_CreateDocument2(DOCID1, strlen(DOCID1), index, NAN, NULL);
+  ASSERT_STREQ(RSLanguage_ToString(d->language), RediSearch_IndexGetLanguage(index));
```

Comment:
Let's add a test that does not use default language, otherwise, we do not know if we took the index default or the global default (English), same for the score...

Reply:
This test does exactly that. The language is `yisddish` and score is `0.42`.

Reply:
ohh right :+1: